pkg/sqlite/load_test.go

```diff
@@ -590,8 +655,9 @@ func TestGetTailFromBundle(t *testing.T) {
 			},
 			expected: expected{
 				err: nil,
-				tail: []string{
-					"csv-c",
+				tail: map[string]tailBundle{
+					"csv-b": {name: "csv-b", channels: []string{"alpha", "stable"}, replaces: []string{"csv-c"}, replacedBy: []string{"csv-a"}},
+					"csv-c": {name: "csv-c", channels: []string{"alpha", "stable"}, replacedBy: []string{"csv-b"}},
```

Comment:
(possibly answered somewhere else that i haven't seen yet...)

Why does the tail contain the requested bundle itself now? In the previous commit, it looks like we start with the bundle replaced by the requested bundle.

Reply:
It does, it was a convenient way to know where to start channel removal from.